vnet/lib/sequel/plugins/ip_address.rb

```diff
@@ -39,12 +40,21 @@ def ipv4_address=(ipv4_address)
             end
             @ipv4_address
           end
+
+          def valid_in_subnet
+            return true unless @network
```

Comment:
This is returned when the IP address does not have @network, in which case it can be ignored. Validate is called again once it does have @network. If it does not get @network it would have failed when validating network id.